oasis.md

```diff
@@ -0,0 +1,67 @@
+External OASIS Repositories
+===========================
+
+We offer hosting of non-OSG CVMFS repositories on OASIS. This means that requests to create, rename, remove, or blanking OASIS repositories will come in as GOC tickets. This document contains instructions for handling those tickets.
+
+!!! note "Also see"
+    * [Policy for OSG Mirroring of External CVMFS repositories](https://opensciencegrid.github.io/technology/policy/external-oasis-repos/)
+    * [External OASIS repository](https://opensciencegrid.github.io/docs/data/external-oasis-repos/)
+
+Hosting a repository on OASIS
+-----------------------------
+
+1.  Ensure that the repository administrator is valid for the VO. This can be done by (a) OSG already having a
+    relationship with the person or (b) the contacting the VO manager to find out.  Review the URL and verify verify
+    that it is appropriate for the VO.
+
+1.  Add the repository URL in OIM under the VO's OASIS repository URLs. This should cause the repository's configuration
+    to be added to the GOC Stratum-0 within 15 minutes.
+
+1.  If the respository ends in a new domain name that has not been distributed before, place a copy of the
+    `domain.name.pub` public key from `domain.name` into `/srv/etc/keys` on both `oasis-replica` and
+    `oasis-replica-itb`. If you do not have that key, ask the repository service representative how to obtain it.  In
+    order to support CVMFS client versions 2.2.X, a symbolic link of `domain.name.conf` has to be made in
```

Comment:
Instead of the passive "has to be",  it would be better to give it as an instruction "make a symbolic link".